DOCUMENTATION.md

```diff
@@ -284,9 +291,73 @@ Revoke support access to an application.
 #### Examples:
     >> > resin.models.application.revoke_support_access('5685')
     'OK'
+## Build
+
+This class implements build model for Resin Python SDK.
+### Function: get(id)
+
+Get a specific build.
+
+#### Args:
+    id (str): build id.
+
+#### Returns:
+    dict: build info.
+
+#### Raises:
+    BuildNotFound: if build couldn't be found.
+### Function: get_all_by_application(app_id, include_logs)
+
+Get a specific build.
```

Comment:
Hmmm, I've never played with the API or either of the SDKs, but this function is named `get_all_by_application`, and it's documented as returning a `list of build info`, so it seems weird that the documentation here says `Get a specific build`?

Also, this function is documented as raising `BuildNotFound: if build couldn't be found`, but looking at the code that doesn't seem to be the case? Is there supposed to be an `ApplicationNotFound` or somehing?

Reply:
Fixed the description and the exception. It won't return an error even I send request to an application that I don't have access so we can remove the exception here.